### DISCUSSION TITLE
电子阳痿了

### DISCUSSION THREAD

**Discussion by Zeteor** [share121/inter-knot#1066] [其他]
现在有了iOS港区账号，有没有好玩的手游推荐一下，最好能用手柄的(„ಡωಡ„)栓Q

**Comment by mhpmc** [comment]
死亡细胞！

**Reply by Zeteor** [reply]
没钱。。。

**Comment by cpu644** [comment]
死亡细胞 几块钱 加一堆dlc 有手柄支持 动作+roguelite
哀恸之日 几块钱 无dlc 我觉得有手柄但是我不知道 rts+剧情向
